prow/plugins.yaml

```diff
@@ -120,11 +120,17 @@ slack:
     - k8s-ci-robot # future home of tide
     - k8s-merge-robot # submit queue
     - k8s-release-robot # anago
-    - wojtek-t # 1.7 patch release manager
-    - jpbetz # 1.8 patch release manager
-    - mbohlool # 1.9 patch release manager
-    - calebamiles # 1.10 branch manager
-    - lavalamp # feature-serverside-apply "branch manager"
+    branch_whitelist:
+        release-1.7:
+            - wojtek-t # 1.7 patch release manager
+        release-1.8:
+            - jpbetz # 1.8 patch release manager
+        release-1.9:
+            - mbohlool # 1.9 patch release manager
+        release-1.10:
+            - calebamiles # 1.10 branch manager
+        feature-serverside-apply:
+            - lavalamp # feature-serverside-apply "branch manager"
```

Comment:
fyi @lavalamp, this should make it reasonable to add more "managers" to `feature-serverside-apply` apply as needed.